resources/settings.xml

```diff
@@ -5,9 +5,10 @@
     <setting id="chromeos_version" value="" visible="false"/>
     <category label="30900"> <!-- Troubleshooting -->
         <setting label="30901" type="action" id="info" action="RunScript(script.module.inputstreamhelper, info)"/>
-        <setting label="30903" help="30904" type="bool" id="disabled" value="false"/>
+        <setting label="30903" help="30904" type="bool" id="disabled" value="false" default="false"/>
```

Comment:
I think the `value="false"` is wrong, it should be replaced with `default="false"`.